services/wallet/responses/router_suggested_routes.go

```diff
@@ -0,0 +1,15 @@
+package responses
+
+import (
+	"github.com/status-im/status-go/errors"
+	"github.com/status-im/status-go/services/wallet/router/routes"
+)
+
+type RouterSuggestedRoutes struct {
+	Uuid                  string                `json:"Uuid"`
+	Best                  routes.Route          `json:"Best,omitempty"`
+	Candidates            routes.Route          `json:"Candidates,omitempty"`
+	TokenPrice            *float64              `json:"TokenPrice,omitempty"`
+	NativeChainTokenPrice *float64              `json:"NativeChainTokenPrice,omitempty"`
+	ErrorResponse         *errors.ErrorResponse `json:"ErrorResponse,omitempty"`
```

Comment:
I think we use camelCase for json everywhere.
```suggestion
	Uuid                  string                `json:"uuid"`
	Best                  routes.Route          `json:"best,omitempty"`
	Candidates            routes.Route          `json:"candidates,omitempty"`
	TokenPrice            *float64              `json:"tokenPrice,omitempty"`
	NativeChainTokenPrice *float64              `json:"nativeChainTokenPrice,omitempty"`
	ErrorResponse         *errors.ErrorResponse `json:"errorResponse,omitempty"`
```

Reply:
In general, the idea of responses is great 👍

Reply:
@igor-sirotin I just kept it as it was, check here https://github.com/status-im/status-go/blob/develop/services/wallet/router/router_v2.go#L156-L163 but since we already have some breaking changes here, maybe better to align on this as well?

Reply:
oh, ok
I would leave it as is then, and prevent breaking changes as much as possible.
It's not that important anyway.